src/screens/styles.ts

```diff
@@ -1,10 +1,54 @@
 import { StyleSheet } from 'react-native';
+import { CurrentRenderContext } from '@react-navigation/native';
 
 export const styles = StyleSheet.create({
   container: {
     flex: 1,
     flexDirection: 'column',
     alignItems: 'center',
     backgroundColor: 'white',
+    padding: 15,
+  },
+  imageContainer: {
+    alignItems: 'center',
+    justifyContent: 'center',
+    marginVertical: 10,
+    marginHorizontal: 15,
+  },
+  profileContainer: {
+    flexDirection: 'row',
+    alignItems: 'center',
+    justifyContent: 'space-around',
+    padding: 10,
+    marginVertical: 10,
+  },
+  username: {
+    marginHorizontal: 10,
+    marginLeft: 25,
+    fontWeight: 'bold',
+  },
```

Comment:
We also want to avoid applying spacing styles (margins/paddings) directly to text. The divs which surround text should determine text positioning.